Source/Login/.docker/config.yaml

```
@@ -4,6 +4,14 @@ serve:
 urls:
   error: http://localhost:8080/error
 
+providers:
+  microsoft:
+    name: Microsoft
+    image_url: https://aadcdn.msauth.net/shared/1.0/content/images/microsoft_logo_ed9c9eb0dce17d752bedea6b5acda6d9.png
+  github:
+    name: GitHub
+    image_url: https://github.githubassets.com/images/modules/logos_page/GitHub-Mark.png
+
```

I think this should be empty by default
```suggestion
providers: {}

```